crates/ruma-appservice-api/src/event/push_events.rs

```diff
@@ -31,6 +33,18 @@ pub mod v1 {
 
             /// A list of events.
             pub events: &'a [Raw<AnyTimelineEvent>],
+
+            #[cfg(feature = "unstable-msc3202")]
+            /// Device lists
+            pub device_lists: DeviceLists,
```

Comment:
It would be nice if the doc comments were more descriptive than the field names. Also, doc comments should come before attributes.

Reply:
Also, all the new fields need `#[serde(default, skip_serializing_if = "FieldType::is_empty")]` because the MSC says

> All the new fields can be omitted if there are no changes for the appservice to handle.

Reply:
Right, of course. They're a bit more fleshed out now, I hope it'll do.